pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx

```diff
@@ -71,16 +72,14 @@ export const KubernetesInventory: FC = () => {
             setAddModalVisible(false);
           }}
           render={({
-            form, handleSubmit, valid, pristine
+            handleSubmit, valid, pristine
           }) => (
             <form onSubmit={handleSubmit}>
               <>
-                <Field
-                  data-qa="kubernetes-cluster-name-field"
+                <TextInputField
                   name="name"
                   label={Messages.kubernetes.addModal.fields.clusterName}
-                  component={InputFieldAdapter}
-                  validate={validators.compose(validators.required)}
+                  validators={[validators.required]}
                 />
                 <Field
```

Comment:
We could replace it with a similar one from the platform-core )

Reply:
I don't know how you are seeing that but I did that already, if you click view file you will see 🤔

Reply:
No, I'm speaking about the TextArea, and it's still there.

Reply:
My excuses, just did 🎉